src/AccessibilityInsights.Extensions/Interfaces/IssueReporting/IssueConfigurationControl.cs

```diff
@@ -9,6 +10,11 @@ namespace AccessibilityInsights.Extensions.Interfaces.IssueReporting
     /// </summary>
     public abstract class IssueConfigurationControl : UserControl
     {
+        /// <summary>
+        ///  To be called to enable the save button in AI-WIN. Will be passed in.
+        /// </summary>
+        public abstract Action UpdateSaveButton { get; set; }
```

Comment:
I think we should remove `CanSave` belown and make this an `Action<bool>` which takes in the new state of the button.

Reply:
Resolving per offline conversation with @flyingUnderTheRadar